admin/sql/create_tables.sql

```diff
@@ -67,7 +67,17 @@ CREATE TABLE revision (
     id          SERIAL      NOT NULL,
     review_id   UUID,
     "timestamp" TIMESTAMP   NOT NULL,
-    text        VARCHAR     NOT NULL
+    text        VARCHAR,     
+    rating      SMALLINT CHECK (rating >= 0 AND rating <= 100)
```

Comment:
I don't think it's a good idea to make the scale "incorrect" from the start. I think it's better to match it with the way it's going to be used.

Migration of ratings from MusicBrainz is not going to happen any time soon. When it does, we'll be able to easily convert ratings to another scale for storage, if that's even needed. But I'd rather keep it the way it's supposed to be until we need to make these changes.